R/helper_starprep.R

```diff
@@ -249,10 +249,22 @@ starprep <- function(...,
                      se_type = NULL,
                      clusters = NULL,
                      alpha = 0.05) {
+
+  fits <- list(...)
+
+  first_object <- fits[[1]]
+
+  gave_list <- length(fits) == 1 & is.list(first_object)
+
+  if(gave_list){
+    is_list_of_lm <- sapply(first_object, inherits, what = c("lm","lm_robust"))
+    if(all(is_list_of_lm)) fits <- first_object
+  }
+
   fitlist <- lapply(
-    list(...),
+    fits,
     function(x) {
-      if (class(x)[1] == "lm")
+      if (inherits(x, "lm"))
```

Comment:
+1 - also what's the point of the `else x` - when would that ever be desired.

Reply:
when x is "lm_robust", no?

Reply:
OK LGTM.